### DISCUSSION TITLE
Can we modify the UI Dashboard ?

### DISCUSSION THREAD

**Discussion by apandey846** [jobrunr/jobrunr#379] [General]
Hi @rdehuyss ,

From Extensible point of view, I read on Jobrunr website that we can modify the Dashboard too ? Want to understand how we can do that as everything we are getting as part of job-runr lib 4.0.9 maven. Any pointers here would be great ?

Actually we are exploring how can we modify the Job payload which was failing coz of some bad data and as its getting retried why not modify that payload from UI and re-enqueue .

Thanks

**Comment by rdehuyss** [comment]
Hi @apandey846 ,

I'm afraid that was originally the plan but I did not have time for it. I'll remove it also from the website with the upcoming release.

In the PRO version, you have Job Migrations to solve this kind of errors.

Kind regards,
R
